### PR TITLE
Added tutorial document for deploying lmdeploy on Jetson series boards. 

### PR DESCRIPTION
**Pull request by BestAnHongjun**
Added tutorial document for deploying lmdeploy on Jetson series boards. 
添加了对Jetson系列板卡部署LMDeploy的教程文档。

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -155,6 +155,10 @@ print(response)
   - [长文本推理](./docs/zh_cn/advance/long_context.md)
   - [多模型推理服务](./docs/zh_cn/serving/proxy_server.md)
 
+# 第三方资源：
```
Comment (lvhan028) [inline README_zh-CN.md:158]: 改为 社区项目 如何？

File: README.md [hunk 2]
```diff
@@ -154,6 +154,10 @@ For detailed user guides and advanced guides, please refer to our [tutorials](ht
   - [Long context inference](docs/en/advance/long_context.md)
   - [Multi-model inference service](docs/en/serving/proxy_server.md)
 
+# Third-party resources
```
Comment (lvhan028) [inline README.md:157]: How about "Third-party projects"?

File: README_zh-CN.md (hunk 1 above)
Comment (BestAnHongjun) [reply README_zh-CN.md:158]: 可！

File: README.md (hunk 2 above)
Comment (BestAnHongjun) [reply README.md:157]: That's great!

File: README_zh-CN.md (hunk 1 above)
Comment (lvhan028) [reply README_zh-CN.md:158]: 那有劳修改下。改好后，我这边可以合入了

Comment (BestAnHongjun) [reply README_zh-CN.md]: 已提交commit

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**